_pages/attributes.md

```diff
@@ -21,11 +25,11 @@ It is important to expect any number of characters in the `(string)` datatype un
 <td markdown="1">
 **UUID**<br /> The user's [universally unique identifier](https://en.wikipedia.org/wiki/Universally_unique_identifier).
 </td>
-<td markdown="1">
-![checkmark][checkmark]
+<td markdown>
```

Comment:
Do we need the `markdown` attributes at all anymore, or could we remove them?

Reply:
ah yeah we should be able to ditch them on the checkmark `td`s at least

Reply:
fixed in https://github.com/18F/identity-dev-docs/pull/449/commits/83b48c592aaf7c081c21be0b08d0291bae0e8bf6